docs/external-contributions/external-contributions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: external-contributions
+title: External Contributions
+hide_title: false
+---
+
+# External Contributions
```

Comment:
If you use `hide_title: false`, then you don't need this

Reply:
Whoops, yep, thanks!